examples/storm-starter/src/jvm/org/apache/storm/starter/BlobStoreAPIWordCountTopology.java

```diff
@@ -38,176 +47,66 @@
-        public void nextTuple() {
-            Utils.sleep(100);
-            _collector.emit(new Values(getRandomSentence()));
-        }
-
-        @Override
-        public void ack(Object id) {
-        }
-
-        @Override
-        public void fail(Object id) {
-        }
-
-        @Override
-        public void declareOutputFields(OutputFieldsDeclarer declarer) {
-            declarer.declare(new Fields("sentence"));
-        }
-
-    }
-
-    // Bolt implementation
-    public static class SplitSentence extends ShellBolt implements IRichBolt {
-
-        public SplitSentence() {
-            super("python", "splitsentence.py");
-        }
-
-        @Override
-        public void declareOutputFields(OutputFieldsDeclarer declarer) {
-            declarer.declare(new Fields("word"));
-        }
-
-        @Override
-        public Map<String, Object> getComponentConfiguration() {
-            return null;
-        }
-    }
-
-    public static class FilterWords extends BaseBasicBolt {
-        boolean poll = false;
-        long pollTime;
-        Set<String> wordSet;
-        @Override
-        public void execute(Tuple tuple, BasicOutputCollector collector) {
-            String word = tuple.getString(0);
-            // Thread Polling every 5 seconds to update the wordSet seconds which is
-            // used in FilterWords bolt to filter the words
-            try {
-                if (!poll) {
-                    wordSet = parseFile(fileName);
-                    pollTime = System.currentTimeMillis();
-                    poll = true;
-                } else {
-                    if ((System.currentTimeMillis() - pollTime) > 5000) {
-                        wordSet = parseFile(fileName);
-                        pollTime = System.currentTimeMillis();
-                    }
-                }
-            } catch (IOException exp) {
-                throw new RuntimeException(exp);
-            }
-            if (wordSet !=null && !wordSet.contains(word)) {
-                collector.emit(new Values(word));
-            }
-        }
-
-        @Override
-        public void declareOutputFields(OutputFieldsDeclarer declarer) {
-            declarer.declare(new Fields("word"));
-        }
-    }
-
-    public void buildAndLaunchWordCountTopology(String[] args) {
-        TopologyBuilder builder = new TopologyBuilder();
-        builder.setSpout("spout", new RandomSentenceSpout(), 5);
-        builder.setBolt("split", new SplitSentence(), 8).shuffleGrouping("spout");
-        builder.setBolt("filter", new FilterWords(), 6).shuffleGrouping("split");
-
-        Config conf = new Config();
-        conf.setDebug(true);
-        try {
-            conf.setNumWorkers(3);
-            StormSubmitter.submitTopologyWithProgressBar(args[0], conf, builder.createTopology());
-        } catch (InvalidTopologyException | AuthorizationException | AlreadyAliveException exp) {
-            throw new RuntimeException(exp);
-        }
-    }
-
```

Comment:
Why are we moving locations of the functions

Reply:
There are checkstyle condition that decide import order, modified overloaded method order and at-clause orders. To avoid these violations, we need to put these methods in right order.